src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -325,10 +325,12 @@
     <WriteLinesToFile Lines="@(_EmccCFlags)" File="$(_EmccCompileRsp)" Overwrite="true" WriteOnlyWhenDifferent="true" />
     <ItemGroup>
       <FileWrites Include="$(_EmccCompileRsp)" />
+      <EmscriptenMinimalBuildEnvVars Include="@(EmscriptenEnvVars)" />
+      <EmscriptenMinimalBuildEnvVars Include="FROZEN_CACHE=" />
```

Comment:
The Microsoft.NET.Runtime.Emscripten.Cache SDK overrides the `WasmCachePath` property.  If we're unfreezing the cache here does that mean we can overwrite the nuget contents sometimes?

Reply:
See https://github.com/dotnet/emsdk/pull/211

Reply:
I think we do that on purpose IIRC https://github.com/dotnet/runtime/blob/9f145a3ec35cf4a9455a011f0195d17b1e65ab94/src/mono/wasm/build/WasmApp.Native.targets#L332-L333 Is that right @radical?

Reply:
the freezing/build was required before we were shipping the cache package we should probably review the logic there

Reply:
to provide more context: the reason for the change itself is that the default value of `FROZEN_CACHE` proably changed with newer emscripten, and so the warming `embuilder build MINIMAL` build was now failing, complaining about not being able to update frozen cache.

Reply:
This whole does need to be reviewed again. If the workload is installed in non-user-writable location then how does the build behave? How does the `FROZEN_CACHE` setting affect things? I have opened https://github.com/dotnet/runtime/issues/83281 .